src/toolbar/components/toolbar-menu/toolbarButton.sample.json

```diff
@@ -16,5 +16,6 @@
   "img_url": "data/some_img.png",
   "explorer": true,
   "url": "reload",
-  "url_parms": "url_parms"
+  "url_parms": "url_parms",
```

Comment:
FWIW: you can just use the ES6 shorthand properties notation: https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand